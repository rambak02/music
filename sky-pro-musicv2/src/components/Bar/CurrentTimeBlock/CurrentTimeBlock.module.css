.timeBlock {
    width: 150px;
    font-style: normal;
    font-weight: 400;
    font-size: 16px;
    line-height: 24px;
    text-align: right;
    color: #ffffff;
    padding: 10px 30px;
    box-sizing: border-box;
}