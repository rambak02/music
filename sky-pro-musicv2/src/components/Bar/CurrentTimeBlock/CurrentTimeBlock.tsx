import styles from "./currentTimeBlock.module.css";
import { formatSecond } from "../helper/format";

type CurrentTimeBlockProps = {
  currentTime: number;
  duration: number;
};

export const CurrentTimeBlock = ({
  currentTime,
  duration,
}: CurrentTimeBlockProps) => {
  const allTimeBar = formatSecond(Number(duration.toFixed(0)));
  const currentTimeBar = formatSecond(Number(currentTime.toFixed(0)));

  return (
    <div
      className={styles.timeBlock}
    >{`${currentTimeBar} / ${allTimeBar}`}</div>
  );
};
