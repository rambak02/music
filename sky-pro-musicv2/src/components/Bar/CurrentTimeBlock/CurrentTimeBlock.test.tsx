import renderer from "react-test-renderer";
import { CurrentTimeBlock } from "./CurrentTimeBlock";

describe("Компонент Navigation", () => {
  it("Должен отрендерить логотип", () => {
    const testResult = renderer
      .create(<CurrentTimeBlock currentTime={2.377663} duration={135.444898} />)
      .toJSON();

    expect(testResult).toMatchSnapshot();
  });
});
