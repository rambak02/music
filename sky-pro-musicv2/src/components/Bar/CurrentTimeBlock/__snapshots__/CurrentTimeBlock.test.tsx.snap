// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Компонент Navigation Должен отрендерить логотип 1`] = `
<div
  className="timeBlock"
>
  00:02 / 02:15
</div>
`;
